src/Link.php

```diff
@@ -64,10 +64,8 @@ public static function create(string $title, \DateTimeInterface $from, \DateTime
     {
         // When creating all day events, we need to be in the UTC timezone as all day events are "floating" based on the user's timezone
         if ($allDay) {
-            $startDate = new \DateTime($from->format('Y-m-d'), new \DateTimeZone('UTC'));
             $numberOfDays = $from->diff($to)->days + 1;
-
-            return self::createAllDay($title, $startDate, $numberOfDays);
+            return self::createAllDay($title, $from, $numberOfDays);
```

Comment:
Why was it dropping the time here? This results in wrong dates down the line I believe.